README.md

```diff
@@ -1,18 +1,18 @@
 # lords-ands-lads
-Rulebook for the Lords &amp; Lads stump variant
+Rulebook for the Lords &amp; Lads stump/hammerschlagen variant
 
 # Rules
 ## You will need:
+* At least 2 players
 * 1 stump
 * 1 hammer
 * 1 nail for each player. These nails must all be the same type
-* 3 nails to be placed in the center of the stump. These nails do not necessarily have to be the same
-* Any number of players
+* Ceiling(n/2 - 1) nails to be placed in the center of the stump when n is the number of players (7 players = 3 nails, 8 players = 4 nails, etc). These must also be the same type
```

Comment:
"You will need at least 2 players"
"Ceiling(n/2 - 1) nails" where n=2
Ceiling(2/2 - 1)
= 0
🤔

Reply:
PEMDAS strikes again

Reply:
8 players = ~4 nails~ **3 nails**
`Ceiling(8/2 - 1) = Ceiling(4 - 1) = Ceiling(3) = 3`

The goal of the balancing was to always have a majority of lads, but never equal numbers, right?

Reply:
Yeah the formula is right it just needs a minimum of 3 players. 2 players doesn't make sense anyway